src/mbgl/annotation/shape_annotation_impl.cpp

```diff
@@ -88,16 +88,15 @@ void ShapeAnnotationImpl::updateTile(const TileID& tileID, AnnotationTile& tile)
         const double tolerance = baseTolerance / (maxAmountOfTiles * GeometryTileFeature::defaultExtent);
 
         geojsonvt::ProjectedRings rings;
-        std::vector<geojsonvt::LonLat> points;
+        mapbox::geometry::linear_ring<double> points;
```

Comment:
Can this be `std::vector<mapbox::geometry::point<double>>` instead? `linear_ring` is not technically correct because this is processing both line and polygon geometry, and `linear_ring` is an appropriate type only for the latter.